feat: massive docs rewrite (#473)
WICG/webmonetization: src/content/docs/support/how-payments-work.mdx

---
title: How payments work
---

import { CodeBlock,Tooltip,LinkOut } from "@interledger/docs-design-system";

## Digital wallets

> To send (and receive) payments, you must have an account with a compatible digital wallet. Not all digital wallets support sending payments with Web Monetization. 

Reviewer: I'll add that in somewhere. I was thinking of staying away from Open Payments for the supporter side of things, but we don't have to.

A digital wallet is the entity that supplies you with the financial account you'll use to make payments. Funding your account typically requires you to link a card or bank to your wallet. 

Since wallet providers are a type of financial entity, they are regulated within the countries they operate. One type of regulation, known as KYC (Know Your Customer), requires such entities to collect personal information and verify your identity before allowing you to open an account.

## Web Monetization-compatible digital wallets

You must have an account with a [compatible digital wallet](/resources/wallets) to send payments. Be sure to select a wallet that's available in your region and supports your preferred currency. 

> Your wallet provider will assign your account a unique ID called either a *payment pointer* or a *wallet address*. This ID is kind of like an account name. It's a publicly shareable ID that you will need when linking your Web Monetization extension to your wallet. 
> 

Reviewer 2: 1. Would an analogy using an email address work better here instead of "like an account name"?
2. I would propose replacing `ID` with `identifier`?
3. Rephrased a bit the last sentence. I feel like `ID`/`identifier` was used too many times in this context.

```suggestion
Your wallet provider will assign your account a unique identifier called either a *payment pointer* or a *wallet address*. This identifier is kind of like an account name. It is publicly shareable and you will need it when linking your Web Monetization extension to your wallet. 
```

Reviewer: Good calls on all.

> The format of the ID depends on whether your wallet provider assigned you a payment pointer or a wallet address. 
> 
> | ID name | Example format |
> | ------- | -------------- |
> | Payment pointer | `$wallet.example.com/alice` |
> | Wallet address | `https://wallet.example.com/alice` |

Reviewer 2: _placeholder: if we replace `ID` with `identifier`_
```suggestion
The format of the identifier depends on whether your wallet provider assigned you a payment pointer or a wallet address. 

| Identifier | Example format |
| ------- | -------------- |
| Payment pointer | `$wallet.example.com/alice` |
| Wallet address | `https://wallet.example.com/alice` |
```
